pkg/dart2native/lib/generate.dart

```diff
@@ -120,6 +122,11 @@ class _Generator {
   /// Specifies the file debugging information should be written to.
   final String? _debugFile;
 
+  /// Specifies the CPU architecture the executable is being generated for. This
+  /// must be provided when [_kind] is [Kind.exe], and it must match the current
+  /// CPU architecture.
+  final String? _targetArch;
```

Comment:
According to Dart Doc, the first line should be brief: https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

Doing as suggested will improve the generated API documentation (`dart doc`) experience.

```suggestion
  /// Specifies the CPU architecture the executable is being generated for. 
  ///
  /// This must be provided when [_kind] is [Kind.exe], and it must match the current
  /// CPU architecture.
  final String? _targetArch;
```

Reply:
Alright, I wasn't aware of that since I based it on `_targetOS` for the doc comment.

Reply:
I've updated the comment.